File: VB/HowToAccessSchedulerControl.Module.Win/WinSchedulerController.vb
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base
Imports DevExpress.ExpressApp.Scheduler.Win
Imports DevExpress.Persistent.Base.General
Imports DevExpress.XtraScheduler

Namespace HowToAccessSchedulerControl.Module.Win
    Public Class WinSchedulerController
        Inherits ObjectViewController(Of ListView, IEvent)

        Protected Overrides Sub OnViewControlsCreated()
            MyBase.OnViewControlsCreated()
            Dim listEditor As SchedulerListEditor = TryCast(View.Editor, SchedulerListEditor)
            If listEditor IsNot Nothing Then
                Dim scheduler As SchedulerControl = listEditor.SchedulerControl
                scheduler.Views.DayView.VisibleTime = New TimeOfDayInterval(New TimeSpan(8, 0, 0), New TimeSpan(17, 0, 0))
            End If
        End Sub
    End Class
End Namespace

File: CS/HowToAccessSchedulerControl.Module.Win/WinSchedulerController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;
using DevExpress.ExpressApp.Scheduler.Win;
using DevExpress.Persistent.Base.General;
using DevExpress.XtraScheduler;

namespace HowToAccessSchedulerControl.Module.Win {
    public class WinSchedulerController : ObjectViewController<ListView, IEvent> {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            SchedulerListEditor listEditor = View.Editor as SchedulerListEditor;
            if (listEditor != null) {
                SchedulerControl scheduler = listEditor.SchedulerControl;
                scheduler.Views.DayView.VisibleTime =
                    new TimeOfDayInterval(new TimeSpan(6, 0, 0), new TimeSpan(11, 0, 0));
            }
        }
    }
}
